packages/main/package-lock.json

```diff
@@ -46,37 +46,6 @@
         "@babel/highlight": "^7.12.13"
       }
     },
-    "@babel/core": {
```

Comment:
Is this correct? I don't see a reason for these changes to modify the deps lock.

Reply:
Not sure. I didn't manually change it.

Reply:
Can you remove this file from the PR then?